docs/ManualDeploymentGuideScalarDLAuditorOnAzure.md

```diff
@@ -0,0 +1,141 @@
+# Deploy Scalar DL Auditor on Azure
+
+This guide shows you how to manually deploy Scalar DL Auditor on a managed database service and a managed Kubernetes service in Azure as part of deploying Scalar DL for production.
+If you have not deployed Scalar DL Ledger, please follow [Deploy Scalar DL on Azure](ManualDeploymentGuideScalarDLOnAzure.md) guide.
+
+## What we create
+
+![image](images/auditor-arch.png)
+
+In this guide, we will create the following components for auditor.
+
+* An Azure Virtual Network associated with a Resource Group.
+* An AKS cluster with 2 node pools.
+* A managed database service.
+* A Cosmos DB Account.
+* A Bastion instance with a public IP.
+* Azure container insights.
+
+## Prerequisites
+
+* Scalar DL Ledger deployment with auditor configuration completed.
+
+## Step 1. Create an environment
+
+Scalar DL Auditor component is implemented to detect Byzantine faults,
+it manages the identical states of Ledger so you need to deploy Ledger and Auditor in separate administrative domains. For more details, please refer to  [Getting Started with Scalar DL Auditor](https://github.com/scalar-labs/scalardl/blob/master/docs/getting-started-auditor.md) guide.
+
+In this guide, for ease of explanation, we deploy Auditor in a different cluster on the same subscription. However, it is highly recommended to deploy it in another administrative domain in production.
+
+### Recommendations
+
+* You should deploy Ledger and Auditor in separate administrative domains.
+
+### Steps
+
+* Follow step 1 to step 3 in the [Deploy Scalar DL on Azure](./ManualDeploymentGuideScalarDLOnAzure.md) guide to create an Scalar DL Auditor environment.
+
+## Step 2. Peer the Virtual Networks
```

Comment:
BTW, I think we should configure security groups (or Azure equivalent ones) between VPCs but it doesn't seem to be set.
Only required ports should be open between VPCs otherwise separating them doesn't seem to mean anything.
Any reason for that?

Reply:
Thank you for your review.
Document updated with the security group (e7bc2c2 and 9ea12d6)